pkg/plugin/aqua/client/client.go

```diff
@@ -30,11 +32,36 @@ func (c *client) newGetRequest(url string) (*http.Request, error) {
 	req.Header.Add("Content-Type", "application/json; charset=UTF-8")
 	req.Header.Add("User-Agent", userAgent)
 	if auth := c.authorization.Basic; auth != nil {
-		req.SetBasicAuth(auth.Username, auth.Password)
+		req.Header.Add(c.authHeader, "Basic "+base64.StdEncoding.EncodeToString([]byte(auth.Username+":"+auth.Password)))
```

Comment:
maybe use formatting instead of concatenating

Reply:
thanks Chen, I have updated this.